de-lib/src/main/java/org/iplantc/de/theme/base/client/collaborators/GroupDisplayStrings.properties

```diff
@@ -1,3 +1,5 @@
+addGroup=Add List
+deleteGroup=Delete List
 deleteGroupConfirm=By deleting the collaborator list {0}, any data that has been shared with this list will immediately be un-shared.  Do you wish to continue deleting the collaborator list?
```

Comment:
I think my view of teams and collaborator lists (coll-lists) is the same as Dennis - the difference is a matter of visibility.

The primary use case for having coll-lists is to be able to share things with a group of users at one time, without having to go through that same list of users every time you want to share things with them. Just like teams, I think it will be very useful to have nested collaborator lists up to say three levels deep.

Unsharing gets a bit more complicated, because I can see a few different use cases.

1) I want to stop sharing with one member of a coll-list:
If I remove them from the list, then I should stop sharing everything that has been shared with the list just for that one user. Likewise, if I add a user to the list, everything that is already shared with that list will be shared with the new user.

2) I want to stop sharing with all members of a list and don't plan to use the list again: 
In this case, the easiest solution would be to delete the list, which would automatically remove sharing permissions for anyone on the list, unless something has also been shared with them individually as well, in which case maybe that permission could remain. Maybe that last part is too difficult to manage.

3) I want to stop sharing some or all of the items shared with a coll-list with all members of the list, but I plan to continue using the list:
In this case, I think I would have to remove sharing privileges for "list" from any item I no longer want the members of the list to have access to.  That would be very tedious if I have shared a lot of things with "list", but I don't see another way to do it. I guess another option would be if there was a  dialog for "show me everything shared with this list (or individual user for that matter) and then I could select items one by one or maybe select all.

Hope that is helpful! If not, bug me.